authorization/role/model/role_scope.go

```diff
@@ -0,0 +1,8 @@
+package model
+
+type RoleScope struct {
```

Comment:
might remove this and end up using the `RoleScope` from `role/scope/repository/role_scope.go`

Reply:
Shouldn't this go in the authorization/role package? i.e. same as [this](https://github.com/fabric8-services/fabric8-auth/blob/master/authorization/organization/organization.go)?